src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +18,12 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(s -> Arrays.stream(s.split(",")))
+                .mapToInt(Integer::parseInt)
+                .filter(n -> n % 2 == 0).min()
+                .orElseThrow(() ->
```

Comment:
```suggestion
                .filter(n -> n % 2 == 0)
                .min()
                .orElseThrow(() ->
```
![image](https://github.com/mate-academy/jv-stream-github-practice/assets/20326858/dd8f8968-b14c-4178-a7e2-54eb5c763253)